src/ray/core_worker/lib/java/jni_utils.h

```diff
@@ -160,6 +160,21 @@ extern jfieldID java_actor_creation_options_group;
 /// bundleIndex field of ActorCreationOptions class
 extern jfieldID java_actor_creation_options_bundle_index;
 
+/// PlacementGroupCreationOptions class
+extern jclass java_placement_group_creation_options_class;
+/// PlacementStrategy class
+extern jclass java_placement_group_creation_options_strategy_class;
+/// global field of PlacementGroupCreationOptions class
+extern jfieldID java_placement_group_creation_options_global;
+/// name field of PlacementGroupCreationOptions class
+extern jfieldID java_placement_group_creation_options_name;
+/// bundles field of PlacementGroupCreationOptions class
+extern jfieldID java_placement_group_creation_options_bundles;
+/// strategy field of PlacementGroupCreationOptions class
+extern jfieldID java_placement_group_creation_options_strategy;
+/// strategy value method of PlacementGroupCreationOptions class
```

Comment:
strategyValue method of PlacementGroupCreationOptions class？

Reply:
Yeah, that's a good point and I have changed it to `value method of PlacementStrategy class` just follow the previous naming